napari/_qt/layer_controls/qt_labels_controls.py

```diff
@@ -369,7 +371,7 @@ def change_contig(self, state):
         state : QCheckBox
             Checkbox indicating if labels are contiguous.
```

Comment:
Not changed here, but the parameter's docstring is related and quite wrong. Here's a hopefully better alternative, but feel free to write your own (or even consider removing the current one): 

```suggestion
        state : int
            Integer value of Qt.CheckState that indicates the check state of contigCheckBox
```